src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.*;
```

Comment:
Wildcard imports can lead to namespace conflicts and make the code less readable. Import only the classes that are needed.